DataHub/Operation/01_ManageETLMappings.md

```diff
@@ -0,0 +1,314 @@
+
+6. If required, select or enter the corresponding configuration settings in the *Configuration* section.   
+
+7. Click the [SAVE] button in the upper right corner.   
+  The changes have been saved. The *Settings* section is hidden.
+
+  > [Info] Be aware that you have to [rerun the mapping](#rerun-an-etl-mapping) to apply the changes made to the attribute.
+
+8. Repeat step **2** to **7** for all attributes you want to map.
+
+
+
+### Add a custom mapping for a destination attribute
+
+When creating an ETL attribute set mapping, you define the language and the channel from which the values are taken for multi-language and multi-channel source attributes.
+Within an attribute set mapping, you can define custom channel and language attribute mappings for multi-language and multi-channel destination attributes.
+
+#### Prerequisites
+
+- At least one ETL attribute set mapping have been created, see [Create an ETL mapping](#create-an-ETL-mapping).
+- The destination attribute is multi-language and/or multi-channel.
+
+#### Procedure
+
+*DataHub > Settings > Tab ETL*
+
+![Attribute set mappings](../../Assets/Screenshots/DataHub/Settings/ETL/AttributeSetMappings.png "[Attribute set mappings]")
+
+1. Click the attribute set mapping you want to add a differing language or channel attribute mapping for a certain destination attribute.   
+  The *Mapping from source set to destination set* view is displayed.
+
+  ![Mapping](../../Assets/Screenshots/DataHub/Settings/ETL/Mapping.png "[Mapping]")
+
+2. Select the checkbox of the attribute mapping to which you want to add a differing mapping for another language or channel.
+  The editing toolbar is displayed above the attribute mappings.
+
+  ![Mapping selected](../../Assets/Screenshots/DataHub/Settings/ETL/MappingSelected.png "[Mapping selected]")
+
+3. Click the [CUSTOM DESTINATION ATTRIBUTE CHANNEL AND LANGUAGE] button in the editing toolbar. The button is only displayed when the destination attribute of the selected mapping is multi-language or multi-channel.
+  The *Settings* section is displayed on the right side.
+
+  ![Custom mapping](../../Assets/Screenshots/DataHub/Settings/ETL/CustomMapping.png "[Custom mapping]")
+
+  > [Info] Depending on the destination attribute, the additional drop-down lists displayed in the *Settings* section differ:
+    - If the destination attribute is multi-language, the *Language* drop-down list is displayed additionally below the destination attribute.
+    - If the destination attribute is multi-channel, the *Channel* drop-down list is displayed additionally below the destination attribute.
+    - If the destination attribute is multi-language and multi-channel, the *Language* and the *Channel* drop-down list are additionally displayed below the destination attribute.
```

Comment:
```suggestion
   > [Info] Depending on the destination attribute, the additional drop-down lists displayed in the *Settings* section differ:  
    - If the destination attribute is multi-language, the *Language* drop-down list is displayed additionally below the destination attribute.  
    - If the destination attribute is multi-channel, the *Channel* drop-down list is displayed additionally below the destination attribute.  
    - If the destination attribute is multi-language and multi-channel, the *Language* and the *Channel* drop-down list are additionally displayed below the destination attribute.  
```

Allg. Doku-Frage: Info unter Bild erlaubt oder lieber nicht? Ich meine, du hast dies in Buchhaltung angemerkt/korrigiert.